backend/tex/aux_parser.py

```diff
@@ -0,0 +1,25 @@
+import argparse
+import json
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--file', help='filepath to input aux')
+args = parser.parse_args()
+
+f = open(args.file, 'r')
+s = '\\newlabel'
+
+listOfLabels = []
+
+for line in f:
+    if line.startswith(s):
+        line = line.replace('}', '')
+        line = line.split('{')
```

Comment:
In fact, what is written to aux file is not that simple. It depends on whether `hyperref` package is used, and proper parsing of `\newlabel` arguments is not as trivial as replacing and splitting on curly braces. See e.g. this http://tex.stackexchange.com/questions/111280/understanding-how-references-and-labels-work

So I suggest sticking to the label only for now, which is always the first argument of `\newlabel`. You can probably search for opening and closing curly bracket starting from `\newlabel` end. 

In subsequent pull requests you can add deep parsing of the second argument.